content.tex

```diff
@@ -24,17 +24,20 @@ \section{Types of Out Of Hours Support}
 
 Remote support will be more common than in-person support.  Remote support can be provided by DevOps, software or hardware personnel.  This document refers to this type of work as remote `On-Call' out of hours support. Compensation is provided for On-Call support for the duration of the On-Call hours assigned and additionally for the hours spent on fault resolution.  Please see the Allowance sections below for more details.
 
-In-person support would entail waking a person up in the case of an emergency and having them go up to the summit to fix a severe problem out of hours. This will be a rare occurrence, and should be avoided unless there is a safety issue with the telescope systems which requires in-person urgent attention.   Additionally, in-person support could be requested when the needs of observing are critical and must move forward and it therefore makes sense to have a person drive up to fix the systems same-night in order for observations to make progress.  The needs of in-person support must be weighted against waiting for the next workday for the day crew to triage and resolve the problem.
+In-person support would entail waking a person up in the case of an emergency and having them go up to the summit to fix a severe problem out of hours. 
+This will be a rare occurrence, and should be avoided unless there is a safety issue with an observatory system requiring immediate in-person attention.
+Additionally, in-person support may be requested when the needs of observing are time-critical and having someone immediately drive up to resolve the issue is worthwhile in order to continue observations.  
+The needs of in-person support must be weighted against waiting for the next workday for the day crew to triage and resolve the problem.
 
 In-person out of hours support may be via an independent vehicle to be on summit as quickly as possible to solve an emergency issue at night.  In-person out of hours support may also be via the Rubin bus system on a day the staff member was not scheduled to arrive at the observatory summit (Saturday, Sunday, holiday) in order to fix an urgent problem.
 
-Some out of hours support could start off as remote support and escalate to in-person support out of hours.  If an issue de-escalates during remote out of hours support and is deemed no longer urgent, then the following day shift is then designated to fixing it, which is not considered out of hours support.
+Some out of hours support could start off as remote support and escalate to in-person support out of hours.  If an issue de-escalates during remote out of hours support and no longer urgent, then the following day shift is then designated to fixing it, which is not considered out of hours support.
```

Comment:
I find this to be quite confusing.
You're saying that work done by the day shift is considered in-hours... but it sounds like if an issue is de-escalated then the out-of-hours work are not specially compensated:
How about:

Some out of hours support tasks may being as remote support and escalate to in-person support. If during the out-of-hours support, the issues is de-escalated and no longer urgent, then the following day shift can be assigned to fix the problem as part of their standard shift.